frontend/src/components/MapView/LeftPanel/LeftPanelTabs.tsx

```diff
@@ -0,0 +1,136 @@
+import { createStyles, makeStyles, Tab, Tabs } from '@material-ui/core';
+import { LayersOutlined, BarChartOutlined } from '@material-ui/icons';
+import React, { useState } from 'react';
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  dir?: string;
```

Comment:
just curious, what is the purpose of this property

Reply:
removed it